packages/babylon-ts-sdk/docs/api/services.md

```
@@ -2741,6 +2993,102 @@ whatever the injected `writeContract` throws
+
+[`WaitForPeginRegistrationDepthParams`](#waitforpeginregistrationdepthparams)
+
+#### Returns
+
+`Promise`\<[`PeginRegistrationDepthResult`](#peginregistrationdepthresult)\>
+
+#### Throws
+
+if no vault is registered on the first poll.
+
+#### Throws
```

markdownlint-cli2 (0.23.2)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)